uploader/views.py

```
@@ -75,3 +75,110 @@ def form_valid(self, form):
+            return self.make_response({"success": True})
+
+        # Update existing media file
+        media_file = os.path.join(settings.MEDIA_ROOT, self.upload.real_path)
+
+        # Store the old file path for later cleanup (if form is saved)
+        old_file_path = None
+        if self.media.media_file:
+            try:
+                old_file_path = self.media.media_file.path
+            except Exception:
+                pass
```

Ruff (0.14.0)

`try`-`except`-`pass` detected, consider logging the exception

(S110)